polygon-digital-carbon/src/RetirementHandler.ts

```diff
@@ -223,6 +226,56 @@ export function handleMossRetirement(event: CarbonOffset): void {
   incrementAccountRetirements(senderAddress)
 }
 
+export function saveCCO2Retirement(event: burnedCO2Token): void {
+  // Don't process zero amount events
+  if (event.params.amount == ZERO_BI) return
+
+  let credit = loadOrCreateCarbonCredit(CCO2_ERC20_CONTRACT, 'CCO2', null)
+
+  // Set up project/default info for Moss "project"
+
+  if (credit.vintage == 1970) {
+    credit.vintage = 2021
```

Comment:
this is a bit of an assumption... CCO2 technically contains a mix of vintages from different projects and since there is no selective retirement you can't know which speicifc project/vintage a given retirement is of

as long as we never show the vintage in the frontend it should be fine, but I'd prefer some kind of explicit placeholder value in the vintage field here...

Reply:
Revised this so the vintage is 1970

Assigning 1970 as the default vintage for all products should be a steady placeholder that can be filtered out on FE etc